### PR TITLE
Drop support for Python 2.6 and 3.2.

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -54,17 +53,6 @@ basepython =
...
-basepython =
-    python2.6
-deps =
-    {[mindeps]deps}
-    ordereddict
-    nose
-commands =
-    nosetests -I persistentregistry -I security
-
 [testenv:coverage]
```
Comment (jamadden) [inline tox.ini]: There's a [testenv:py26] below this that can go.

File: .travis.yml [hunk 2]
```diff
@@ -7,12 +7,9 @@ matrix:
     - python: 3.5
       env: TOXENV=py35-pure
 env:
-    - TOXENV=py26
-    - TOXENV=py26-minimal
```
Comment (jamadden) [inline .travis.yml]: Should that be replaced with a py27-minimal?

File: tox.ini (hunk 1 above)
Comment (tseaver) [inline tox.ini]: 1fee42a

File: .travis.yml (hunk 2 above)
Comment (tseaver) [inline .travis.yml]: 1fee42a

**Comment by jamadden** [comment]
LGTM pending CI

**Pull request merged**